crepecake/src/main/java/com/google/cloud/tools/crepecake/blob/BlobStream.java

```diff
@@ -31,7 +31,7 @@ public BlobStream() {
     byteArrayOutputStream = new ByteArrayOutputStream(0);
   }
 
-  /** Initializes with the contents of the input blob. */
+  /** Initializes with the contents of the {@param inputStream}. */
```

Comment:
I don't think this is how you use @param in Javadoc.

Reply:
Is there a correct way to refer to the name of the parameter or should I just say `{@code inputStream} parameter`?

Reply:
@code or @link, I think